zendframework/zendframework1/ZF2015-04.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Potential CRLF injection attacks in mail and HTTP headers
+link:      http://framework.zend.com/security/advisory/ZF2015-04
+cve:       CVE-2015-3154
+branches:
+    1.12.x:
+        time:     2015-05-19 17:40:42
+        versions: [>=1.12.0,<1.12.12]
+reference: composer://zendframework/zendframework
```

Comment:
Le 20/05/2015 02:49, Christophe Coevoet a écrit :

>> +reference: composer://zendframework/zendframework
> 
> this looks wrong to me. It should be ``composer://zendframework/zendframework1``

You’re right, thanks. Commit updated.